src/app/portlets/content-types/main/content-types.component.ts

```diff
@@ -81,17 +82,18 @@ export class ContentTypesPortletComponent implements OnInit {
     ) {}
 
     ngOnInit() {
-
         observableForkJoin(
             this.dotMessageService.getMessages(this.i18nKeys),
             this.dotContentTypeService.getAllContentTypes(),
             this.dotLicenseService.isEnterprise(),
-            this.pushPublishService
-                .getEnvironments()
-                .pipe(
-                    map((environments: DotEnvironment[]) => !!environments.length),
-                    take(1)
-                )
+            this.pushPublishService.getEnvironments().pipe(
+                map((environments: DotEnvironment[]) => !!environments.length),
+                take(1)
+            ),
+            this.route.data.pipe(
+                pluck('filterBy'),
+                take(1)
+            )
         ).subscribe((res) => {
```

Comment:
```suggestion
        ).subscribe(([var, var, var]) => {
```

Deconstruct the array with names so is easier to understand.